src/test/scala/chiselTests/DPISpec.scala

```diff
@@ -17,6 +17,7 @@ private object EmitDPIImplementation {
     val dpiImpl = s"""
                      |#include <stdint.h>
                      |#include <iostream>
+                     |#include <svdpi.h>
```

Comment:
To reiterate an earlier point: Let's add a prerequisites section near the top. Let's make sure to name what library(ies?) are needed for Chisel DPI, and let's be sure to include the `svdpi.h` header in our examples, since otherwise, the code doesn't work.

Reply:
Let's also link to the documentation so that Chisel users can quickly cross-reference the DPI primitives, such as `svBitVecVal` or `svOpenArrayHandle`.